tsweb/tsweb.go

```diff
@@ -146,8 +146,10 @@ type Port80Handler struct {
 }
 
 func (h Port80Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	ipStr, _, _ := net.SplitHostPort(r.RemoteAddr)
+	ip, _ := netip.ParseAddr(ipStr)
```

Comment:
nit: you could also `ap, _ := netip.ParseAddrPort(r.RemoteAddr)` and use `ap.Addr().IsLoopback()` in the condition